### PR TITLE
- making `string_view(value_type *)` and `string_view(value_type const *)` implicit

### PR DESCRIPTION
**Pull request by juchem**

### CODE REVIEW

File: fatal/string/string_view.h [hunk 1]
```diff
@@ -74,14 +74,14 @@ struct string_view {
 #   endif // __cplusplus > 201400
   }
 
-  explicit string_view(value_type *s):
+  string_view(value_type *s):
```
Comment (yfeldblum) [inline fatal/string/string_view.h]: The internal linter wants explicit `/* implicit */` annotations.

File: fatal/string/string_view.h [hunk 2]
```diff
@@ -74,14 +74,14 @@ struct string_view {
...
     begin_(s),
     end_(std::next(s, std::strlen(s)))
   {
     assert(begin_ <= end_);
   }
 
-  explicit string_view(value_type const *s):
+  string_view(value_type const *s):
```
Comment (yfeldblum) [inline fatal/string/string_view.h]: Likewise.

File: fatal/type/remove_rvalue_reference.h [hunk 3]
```diff
@@ -45,6 +45,9 @@ struct remove_rvalue_reference<T &&> {
   using type = T;
 };
 
+template <typename T>
+using remove_rvalue_reference_t = typename remove_rvalue_reference<T>::type;
```
Comment (yfeldblum) [inline fatal/type/remove_rvalue_reference.h]: Intended? Useful, but seems like a separate PR.

File: fatal/string/string_view.h [hunk 4]
```diff
@@ -74,14 +74,14 @@ struct string_view {
 #   endif // __cplusplus > 201400
   }
 
-  explicit string_view(value_type *s):
+  /* implicit */ string_view(value_type *s):
```
Comment (yfeldblum) [inline fatal/string/string_view.h]: 👍

File: fatal/type/remove_rvalue_reference.h (hunk 3 above)
Comment (juchem) [reply fatal/type/remove_rvalue_reference.h]: leftovers

**Pull request closed without merging**